packages/integrations/markdoc/package.json

```diff
@@ -61,6 +61,7 @@
     "linkedom": "^0.14.12",
     "mocha": "^9.2.2",
     "rollup": "^3.20.1",
+    "shiki": "^0.14.1",
```

Comment:
Shouldn't we add `shiki` as a peer dependency instead? And made it optional via `peerDependenciesMeta`?

Reply:
Ah ha, sounds right to me! Will update